docs/user/helper.rst

```diff
@@ -48,6 +48,53 @@ argument ``idx=0``, the default) and one for y (with argument
     gs.fit(X_sl, y_sl)
 
 
+AccelerateMixin
+---------------
+
+This mixin class can be used to add support for huggingface accelerate_ to
+skorch. E.g., this allows you to use mixed precision training (AMP), multi-GPU
+training, or training with a TPU. For the time being, this feature should be
+considered experimental.
+
+To use this feature, create a new subclass of the neural net class you want to
+use and inherit from the mixin class. E.g., if you want to use a
+:class:`.NeuralNet`, it would look like this:
+
+.. code:: python
+
+    from skorch import NeuralNet
+    from skorch.helper import AccelerateMixin
+
+    class AcceleratedNet(AccelerateMixin, NeuralNet):
+        """NeuralNetClassifier with accelerate support"""
+
+The same would work for :class:`.NeuralNetClassifier`,
+:class:`.NeuralNetRegressor`, etc. Then pass an instance of Accelerator_ with
+the desired parameters and you're good to go:
+
+.. code:: python
+
+    from accelerate import Accelerator
+
+    accelerator = Accelerator(...)
+    net = AcceleratedNet(
+        MyModule,
+        accelerator=accelerator,
+        device=None,
+        callbacks__print_log__sink=accelerator.print)
```

Comment:
Should we do this automatically? This feels like more boilerplate for a user to write.

Reply:
Yes, true. On the other hand, since the accelerator does not exist when we define `AccelerateMixin`, we cannot set it as default argument. We would need to do something else, like:

```python
def initialize(self):
    super().initialize()
    self.set_params(callbacks__print_log__sink=self.accelerator.print)
    return self
```

Not sure if the user would have any reason _not_ to set this, but if they had, this hard-coding would prove annoying.

Reply:
I think having good defaults is generally better. If a user does not want to set it like this, I'm guessing they are more advanced and would need to to override `initialize` to undo it.

It's a toss up for me.